content/en/platform/corda/4.5/enterprise/app-upgrade-notes.md

```diff
@@ -94,8 +94,8 @@ The issue here is that a new `Destination` interface introduced in Platform Vers
 used as an `AbstractParty` but has an actual value that is one of `Party` or `AnonymousParty`. These subclasses
 implement `Destination`, while the superclass does not. Kotlin must pick a type for the variable, and so chooses the most specific
 ancestor of both `AbstractParty` and `Destination`. This is `Any`, which is not a valid type for use as an `AbstractParty` later.
-(For more information on `Destination`, see the [Changelog](https://docs.corda.net/docs/corda-os/4.4/changelog.html) for Platform Version 5, or the KDocs for the interface
-[here](https://docs.corda.net/head/api/kotlin/corda/net.corda.core.flows/-destination.html))
+(For more information on `Destination`, see the [Changelog](https://docs.r3.com/en/platform/corda/4.4/open-source/changelog.html for Platform Version 5, or the KDocs for the interface
```

Comment:
```suggestion
(For more information on `Destination`, see the [Changelog](../../../../../../en/platform/corda/4.4/open-source/changelog.md for Platform Version 5, or the KDocs for the interface
```
We're trying to stay away from absolute links. Think this is what it should be, but please check.